src/app/beer_garden/db/api.py

```diff
@@ -14,17 +14,14 @@
 replace_commands = beer_garden.db.mongo.api.replace_commands
 
 
-@publish_event(Events.DB_CREATE)
```

Comment:
If we're going to remove these decorators then we can remove these function defs and just assign the mongo-specific functions to the api ones (like above)

Reply:
I removed the function definitions.